index.js

```diff
@@ -149,16 +149,17 @@ function initDb(url, data) {
         const collection = db.collection(col)
         return collection.insert(data[col])
       })
-      return Promise.all(requests)
+      return Promise.all(requests).then(() => db.close())
     })
 }
 
 function dropDb(url) {
   return client.connect(url)
-    .then(db => db.collections())
-    .then(collections => {
-      const requests = collections.map(col => col.drop())
-      return Promise.all(requests)
+    .then(db => {
+      return db.collections().then(collections => {
+        const requests = collections.map(col => col.drop())
+        return Promise.all(requests).then(() => db.close())
```

Comment:
this line in not in right place

Reply:
I'm not sure I understand, it is aligned with the previous line

Reply:
maybe like this
```
function dropDb(url) {
  return client.connect(url)
    .then(db => {
      return db.collections()
          .then(collections => {
              const requests = collections.map(col => col.drop())
              return Promise.all(requests)
          })
         .then(() => db.close())
    })
}
```

Reply:
lol same thing but sure